src/main/java/com/esri/arcgisruntime/toolkit/BookmarksList.java

```diff
@@ -0,0 +1,80 @@
+package com.esri.arcgisruntime.toolkit;
+
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.Bookmark;
+import com.esri.arcgisruntime.mapping.BookmarkList;
+import com.esri.arcgisruntime.mapping.view.GeoView;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import javafx.application.Platform;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.util.Callback;
+
+/**
+ * A list of bookmarks from a map or scene. Selecting a bookmark item set's the geoView's
+ * viewpoint to the selected bookmark's viewpoint.
+ */
+public class BookmarksList extends ListView<Bookmark> {
```

Comment:
Should this extend ListView or just have a default ListViewSkin so we can support other skins later? Didn't want to overcomplicate such a simple component.